crates/rpc/rpc-types/src/otterscan.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{Block, Transaction, TransactionReceipt};
+use reth_primitives::{Address, Bytes};
+use serde::{Deserialize, Serialize};
+
+/// Operation type enum for `InternalOperation` struct
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub enum OperationType {
+    /// Operation Transfer
+    OpTransfer,
+    /// Operation Contract self destruct
+    OpSelfDestruct,
+    /// Operation Create
+    OpCreate,
+    /// Operation Create2
+    OpCreate2,
+}
+
+/// Custom struct for otterscan `getInternalOperations` RPC response
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct InternalOperation {
+    r#type: OperationType,
+    from: Address,
+    to: Address,
+    value: u128,
+}
+
+/// Custom struct for otterscan `traceTransaction` RPC response
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct TraceEntry {
+    r#type: String,
+    depth: u32,
+    from: Address,
+    to: Address,
+    value: u128,
+    input: Bytes,
+}
+
+/// Internal issuance struct for `BlockDetails` struct
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct InternalIssuance {
+    block_reward: String,
+    uncle_reward: String,
+    issuance: String,
+}
+
+/// Custom struct for otterscan `getBlockDetails` RPC response
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct BlockDetails {
+    block: Block,
```

Comment:
I didn't notice this at first, but we need a custom serializer (or an entirely different struct even) for this block

e.g.: `ots_getBlockDetails` does not want the `transactions` attribute, and instead wants a `transactionCount` (inside the Block, not as another BlockDetails attribute, as I had understood initially)